raiwidgets/raiwidgets/cohort.py

```diff
@@ -397,6 +398,75 @@ def __init__(self, name: str):
         self.name = name
         self.cohort_filter_list = None
 
+    @staticmethod
+    def _cohort_serializer(obj):
+        """The function to serialize the Cohort class object.
+
+        :param obj: Any member of the Cohort class object.
+        :type: Any
+        :return: Python dictionary.
+        :rtype: dict[Any, Any]
+        """
+        return obj.__dict__
+
+    def to_json(self):
+        """Returns a serialized JSON string for the Cohort object.
+
+        :return: The JSON string for the cohort.
+        :rtype: str
+        """
+        return json.dumps(self, default=Cohort._cohort_serializer)
+
+    @staticmethod
+    def _get_cohort_object(json_dict):
+        """Method to read a JSON dictionary and return a Cohort object.
+
+        :param json_dict: JSON dictionary containing cohort data.
+        :type: dict[str, str]
+        :return: The Cohort object.
+        :rtype: Cohort
+        """
+        if "name" not in json_dict:
+            raise UserConfigValidationException(
+                "No name field found for cohort deserialization")
+        if "cohort_filter_list" not in json_dict:
```

Comment:
I think you can just have a helper method defined which does this check and throws the error, and pass in the "name" and "cohort_filter_list" fields to it.  Or, you can just do:
```
fields = ['name', 'cohort_filter_list']

for field in fields:
    if field not in json_dict:
        raise UserConfigValidationException("No {0} field found for cohort deserialization".format(field))
```

Reply:
Made the change